### ISSUE TITLE
PinSound Not Workimg

### ISSUE THREAD

**Issue by kapter316** [vpinball/vpinball#1347]
Hello, when running PinSound in any version of 10.8, currenlty using beta 8 (x64/x86 GL/NonGL) PinSound will not recognize VPX. However, when I run it in 10.7, it recognizes and plays perfectly. I saw someone back in August open a bug up for this, but it was closed because someone said they believed PinMame controlled this and not VPX. If that is correct, then why does 10.7 with the same setup as 10.8 work for PinSound? Any thoughts? I would be happy to share any additional information that you may need. 
Thank you,
Brian

**Comment by kapter316** [comment]
Some additional information, as now I don't know if this is a VPX or Pinmame issue: 
I downloaded the latest: sc-3.6-720-e4df859-win-x86.zip and sc-3.6-720-e4df859-win-x64.zip. When running x64 test, PinSound works with no problems; you cannot get it to load with VPX10.8.8 32bits launcher. However, when I run x86 through a test or VPX10.8.8 64bits launcher, I always get the following error: "unsupported pinball sound system." I have tried to run as admin and not as admin for all the programs above, and I still have the same issue. Is there a version of pinmame that runs PinSound in VPX10.8 64bits launcher? 
One last piece of information, the 10.7 32bits launcher will load and does play PinSound.
I appreciate everything you do for us.
